build.gradle

```diff
@@ -0,0 +1,34 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'com.android.tools.build:gradle:1.3.0'
+  }
+}
+
+allprojects {
+  repositories {
+    mavenCentral()
+  }
+}
+
+ext {
+  minSdkVersion = 15
```

Comment:
That is probably true, but the cost of supporting 14 is just flipping a bit, so there hasn't been a compelling reason to not support it.